plenum/server/propagator.py

```diff
@@ -81,6 +82,15 @@ def forwarded(self, req: Request) -> bool:
         """
         return self[req.key].forwarded
 
+    def ordered_by_replica(self, request_key):
```

Comment:
Should we call this `mark_as_ordered` to be aligned with `mark_as_executed` and `mark_as_forwarded`?